doc/developer-guides/hld/mmio-dev-passthrough.rst

```diff
@@ -0,0 +1,34 @@
+.. _mmio-device-passthrough:
```

Comment:
This document needs to be added to the table of contents, in hld-hypervisor.rst.  But I actually think this shouldn't be a separate document, and instead be a chapter in the hv-dev-passthrough.rst document.  I'd recommend you move this material there.

Reply:
@dbkinder Thanks a lot for your comments. Will do the modifications as your suggestions except for this one.  We should move this material into hv-dev-passthrough.rst, but we could not just move it there.  The whole picture of hv-dev-passthrough.rst is written for PCI device passthrough from the beginning.  If we want to do this, we need do more effects to achieve this. I don't know if we should do this now.  IE, we could use #BDF to identify the SID of the  PCIdevice, but we have not figured out how to identidy the SID of the MMIO device.